File: web-api-doc/src/main/java/net/yan100/compose/webapidoc/properties/JwtHeaderInfoProperties.java
package net.yan100.compose.webapidoc.properties;

import lombok.Data;
import net.yan100.compose.core.http.Headers;

@Data
public class JwtHeaderInfoProperties {
  private String authTokenName = Headers.AUTHORIZATION;
  private String refreshTokenName = Headers.X_REFRESH;
}

File: depend/springdoc-openapi/src/main/kotlin/io/github/truenine/composeserver/depend/springdocopenapi/properties/JwtHeaderInfoProperties.kt
package io.github.truenine.composeserver.depend.springdocopenapi.properties

import io.github.truenine.composeserver.consts.IHeaders

class JwtHeaderInfoProperties {
  var authTokenName = IHeaders.AUTHORIZATION
  var refreshTokenName = IHeaders.X_REFRESH
}
